Storage/src/Bucket.php

```diff
@@ -267,6 +267,18 @@ public function exists(array $options = [])
      *           Acceptable values include, `"authenticatedRead"`,
      *           `"bucketOwnerFullControl"`, `"bucketOwnerRead"`, `"private"`,
      *           `"projectPrivate"`, and `"publicRead"`.
+     *     @type array $retention Object retention configuration.
```

Comment:
nit: to be consistent with the similar `$metadata` entry below:
```suggestion
     *     @type array $retention The full list of available options are outlined
     *           at the [JSON API docs](https://cloud.google.com/storage/docs/json_api/v1/objects/insert#request-body).
```